azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKInitialization.kt

```diff
@@ -112,9 +137,92 @@ internal class CallingSDKInitialization(
     }
 
     fun dispose() {
+        incomingCall?.removeOnCallEndedListener(onIncomingCallEnded)
+        incomingCall = null
+        callAgentCompletableFuture?.get()?.removeOnIncomingCallListener(callCompositeIncomingCallListener)
         callAgentCompletableFuture?.get()?.dispose()
         callAgentCompletableFuture = null
         callClientInternal = null
         callClientCompletableFuture = null
     }
+
+    fun rejectIncomingCall(callID: String): CompletableFuture<Void> {
```

Comment:
We need to review responsibilities devision between `CallingSDKInitialization` and `CallingSDKWrapper`.
`Initialization` suggest that it will only be responsible to initialize the SDK Wrapper, but then it is getting filled with bunch of logic like `rejectIncomingCall`

Reply:
As per discussion, lets think more about it and sync before merging to develop